injected/src/content-scope-features.js

```diff
@@ -44,6 +45,14 @@ export function load(args) {
 
     const bundledFeatureNames = typeof importConfig.injectName === 'string' ? platformSupport[importConfig.injectName] : [];
 
+    // Initialize detectors early so they're available when features init
```

Comment:
I don't think we should put it that deep into the framework. Can we not trigger it in 'load' and whomever triggers firsts kicks it off for both?